dev/index.ts

```diff
@@ -38,8 +38,12 @@ trackSession((session) => {
     loginBanner.appendChild(logoutButton);
   }
 });
-
-const webIdToShow = "https://angelo.veltens.org/profile/card#me";
+// https://testingsolidos.solidcommunity.net/profile/card#me
+// https://timbl.inrupt.net/profile/card#me
+//
+// const webIdToShow = "https://angelo.veltens.org/profile/card#me";
+const webIdToShow = "https://testingsolidos.solidcommunity.net/profile/card#me";
+// const webIdToShow = "https://timbl.inrupt.net/profile/card#me";
```

Comment:
should we take out all the commented webIdsToShow?  Or are we just leaving them so it's easier when we are testing.